distributor-api-v1/operations.md

```diff
@@ -32,24 +32,30 @@ Preferred initial call used to obtain all static data about distributor configur
             "Id": "9044b0bf-cbe0-4df5-beeb-b32e19bcd073",
             "ImageId": "e956201e-ba2f-470f-8070-b43f9cd72194",
             "Name": {
-                "ru-RU": "Амстердам",
                 "en-US": "Amsterdam"
-            },
-        },
+            }
+        }
     ],
     "CityId": "9044b0bf-cbe0-4df5-beeb-b32e19bcd073",
     "Configurations": [
         {
+            "Id": "3edbe1b4-6739-40b7-81b3-d369d9469c48",
             "AdultCount": null,
             "ChildCount": null,
             "ChildSelectionEnabled": null,
+            "CompetitorRateDescription": {
+                    "en-US": "Comptetitor Rate Description"
+                },
+            "CompetitorPriceRelativeAdjustment": "110",
```

Comment:
this is not a string, it's decimal.
Also, I think the value is incorrect, should not be there 0.5?

Reply:
You are right, I've changed it. It goes over 1, so I've kept the value, but moved decimals.